etc/contracts-test-data/contracts/loadnext/query_read_metrics_basic.sql

```diff
@@ -0,0 +1,43 @@
+-- calculate distribution of storage reads per transaction
+-- does not calculate hot/cold reads
+
+\set start_from_miniblock_number 40000000
+\set miniblock_range 10000
+
+with mb as (select *
+            from miniblocks mb
+            where mb.number >= :start_from_miniblock_number
+            order by mb.number
+            limit :miniblock_range)
+select stddev_samp(metric)                                  as stddev,
+       avg(metric)                                          as avg,
+       sum(metric)                                          as sum,
+       percentile_cont(0.00) within group (order by metric) as pct_00,
+       percentile_cont(0.01) within group (order by metric) as pct_01,
+       percentile_cont(0.05) within group (order by metric) as pct_05,
+       percentile_cont(0.25) within group (order by metric) as pct_25,
+       percentile_cont(0.50) within group (order by metric) as pct_50,
+       percentile_cont(0.75) within group (order by metric) as pct_75,
+       percentile_cont(0.95) within group (order by metric) as pct_95,
+       percentile_cont(0.99) within group (order by metric) as pct_99,
+       percentile_cont(1.00) within group (order by metric) as pct_100
+from (select miniblock_number,
+             (sum(read_write_logs) - sum(write_logs)) / sum(transaction_count) as metric,
```

Comment:
I would appreciate a second set of eyes on this particular calculation

Reply:
I think it's correct